content/blog/ingest-multiple-tables-using-hudi/index.html

```diff
@@ -0,0 +1,233 @@
+<!doctype html>
```

Comment:
Hi, It seems we do not need the *.html file. You can refer to this PR #1992

Reply:
yeah, I don't think you need this file.

Reply:
Thank you for pointing the PR. Removed the file now.